components/support/support-additional.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "id": 20000,
+    "name": "MoonMail",
+    "username": "moonmail",
+    "tier": "sponsor",
+    "avatar": "https://static.moonmail.io/moonmail-logo.svg",
```

Comment:
Would it be better to host the svgs on webpack site instead? I can see pros/cons to both and we can stick with the current solution.